pkg/addon/configpolicy/manifests/managedclusterchart/templates/deployment.yaml

```diff
@@ -51,8 +51,9 @@ spec:
           - --client-max-qps={{ .Values.args.clientQPS }}
           - --client-burst={{ .Values.args.clientBurst }}
           - --health-probe-bind-address=:8081
-          {{- if and .Values.prometheus.enabled (ne .Values.kubernetesDistribution "OpenShift") }}
-          - --metrics-bind-address=0.0.0.0:8383
+          {{- if and .Values.prometheus.enabled (eq .Values.kubernetesDistribution "OpenShift") }}
+          - --secure-metrics=true
+          - --metrics-bind-address=0.0.0.0:8443
```

Comment:
Is this missing an `else if .Values.prometheus.enabled` for port 8383 or is that not necessary?

Reply:
I was thinking that since `config-policy-controller` and `governance-policy-framework` both defaults to port `8383` that it wouldn't be necessary, but looking at this now I realized it should probably override the address to use `0.0.0.0` instead of `localhost` anyways. Thanks for catching this!